variables.tf

```diff
@@ -223,3 +223,9 @@ variable "disk_support_config" {
   })
   default = {}
 }
+
+variable "disk_setup_image" {
+  description = "Docker image for the disk setup script"
+  type        = string
+  default     = "materialize/ephemeral-storage-setup-image:v0.1.0"
```

Comment:
This image still does not exist. To test it out at the moment you can use:

```
  default     = "bobbyiliev/ephemeral-storage-setup-image:v0.1.1"
```

Reply:
I'm not sure what's wrong, but the image still does not exist. It looks like https://github.com/MaterializeInc/ephemeral-storage-setup-image/actions/runs/14566949646/job/40857639988 thinks it pushed it, but I can't pull it or find it on docker hub.

Reply:
Yes, same for me. I think that we need to manually create the repository. I've posted the request in the operations channel.